packages/passport-interface/src/FrogCrypto.ts

```diff
@@ -16,3 +16,9 @@ export interface FrogCryptoFrogData {
   beauty_min: number | undefined;
   beauty_max: number | undefined;
 }
+
+export interface FrogCryptoScore {
+  telegram_username?: string;
```

Comment:
are you sure this is `undefined` in the case a conversation isn't present, or is this actually `null`?

Reply:
you are right :D added a test case and updated